src/structure_new/src/nonlinear_solver/4C_structure_new_nln_solver_generic.cpp

```diff
@@ -21,14 +21,12 @@ Solid::Nln::SOLVER::Generic::Generic(
     const std::shared_ptr<Solid::TimeInt::BaseDataGlobalState>& gstate,
     const std::shared_ptr<Solid::TimeInt::BaseDataSDyn>& sdyn,
     const std::shared_ptr<Solid::TimeInt::NoxInterface>& noxinterface,
-    const std::shared_ptr<Solid::Integrator>& integrator,
+    const std::shared_ptr<Solid::Integrator>& intgr,
```

Comment:
What's the benefit of this change? The new abbreviation is less descriptive than the original.

Reply:
Just to avoid name conflict with with the method `integrator()`, which previously was not an issue, since initialization was in a separate finction, but now I have moved it to ctor. I did not want to use `this->integrator()` since `this` is not frequently used across the code in general (and I agree with this).